src/control/drpc/property.go

```diff
@@ -45,3 +48,12 @@ const (
 	// PoolSelfHealingAutoRebuild sets the self-healing strategy to auto-rebuild.
 	PoolSelfHealingAutoRebuild = C.DAOS_SELF_HEAL_AUTO_REBUILD
 )
+
+const (
```

Comment:
convert to named const e.g. PoolPolicy PoolPolicyDefault = C.TIER_POLICY_DEFAULT then ParsePolicy(string) (PoolPolicy, error) can map string to typed const if found